### PR TITLE
Add/crud for admins 3 31

### LINKED ISSUE #31: Crud for Admins
- [x] add
 - [x] list all
Referencing #17

### PR DESCRIPTION
**Pull request by nitrajka**
Resolves #31

### CODE REVIEW

File: src/src/AdminBundle/Controller/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        $encoded = $this->get('security.password_encoder')->encodePassword($admin, $password);
+        $admin->setPassword($encoded);
+    }
+
+	/**
+	 * @Route("/admins/create", name="create")
+	 */
+	public function createAdmin(Request $request) {
+
+		if ($request->isMethod('POST')) { // ak som submitla noveho admina
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Controller/AdminController.php]: medzera vs tabulator :-(

File: src/src/AdminBundle/Controller/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class AdminController extends Controller
+{
+	/**
+	 * @Route("/admins", name="admins")
+	 */
+	public function indexAction()
+	{
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Controller/AdminController.php]: dohodli sme sa, ze `{` patri na riadok vyssie, zmen to prosim vsade v subore.

File: src/src/AdminBundle/Controller/AdminController.php (hunk 1 above)
Comment (ZoltanOnody) [reply src/src/AdminBundle/Controller/AdminController.php]: rozdel to prosim na 2 routy, jedna nech handluje `POST` druha `GET`

File: src/src/AdminBundle/Controller/AdminController.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+	/**
+	 * @Route("/admins/create", name="create")
+	 */
+	public function createAdmin(Request $request) {
+
+		if ($request->isMethod('POST')) { // ak som submitla noveho admina
+            $email = $request->get('email');
+            $password = $request->get('password');
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Controller/AdminController.php]: takto to prosim nerob. https://symfony.com/doc/3.3/form/without_class.html

File: src/src/AdminBundle/Resources/views/Admin/create.html.twig [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                </div>
+                <div class="form-group row">
+                    <div class="col-sm-8 offset-sm-4">
+                        <button type="submit" class="btn btn-primary">Create admin</button>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </div>
+{% endblock %}
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Resources/views/Admin/create.html.twig]: newline

File: src/src/AdminBundle/Resources/views/Admin/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                    Are you sure you want to delete it?
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-danger">Delete</button>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```
Comment (ZoltanOnody) [inline src/src/AdminBundle/Resources/views/Admin/index.html.twig]: newline

**Review by ZoltanOnody — CHANGES_REQUESTED** [review]

**Review by ZoltanOnody — APPROVED** [review]

**Review by ZoltanOnody — CHANGES_REQUESTED** [review]
dont merge

**Review by Radcho — APPROVED** [review]

**Review by ZoltanOnody — APPROVED** [review]

**Pull request merged**